reference_test.go

```diff
@@ -224,6 +224,29 @@ func TestReferenceIsValidName(t *testing.T) {
 	}
 }
 
+func TestReferenceNormalizeName(t *testing.T) {
+	t.Parallel()
+
+	ref, err := ReferenceNormalizeName("refs/heads//master", ReferenceFormatNormal)
+	checkFatal(t, err)
+
+	if ref != "refs/heads/master" {
+		t.Errorf("refs/heads//master should be normalized correctly")
```

Comment:
Go prefers to print the [want and got](https://golang.org/pkg/testing/) when failing tests:

```suggestion
		t.Errorf("ReferenceNormalizeName(%q) = %q; want %q", "refs/heads//master", ref, want)
```

Same in L241.